.github/workflows/pre-commit.yaml

```diff
@@ -0,0 +1,29 @@
+---
+name: "Install Yamls precommit checks"
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+
+jobs:
+  pre_commit:
+    name: "Pre commit"
+    runs-on: ubuntu-latest
```

Comment:
I think it may be better, just to be safe, to pin the node we are going to use to a specific Ubuntu version.
After all, "latest" is a moving target and tests should be as stable as possible.

Reply:
Since we are moving this job to the prow, so this will be removed.